.env.example

```diff
@@ -0,0 +1,5 @@
+POSTGRES_USER=
+POSTGRES_DATABASE=
+POSTGRES_PASSWORD=
+POSTGRES_HOST=
+POSTRES_PORT=
```

Comment:
@ra0x3 - will values be placed here eventually?

Reply:
@rfuelsh Yea the idea is that .env.example will show configuration (primarily for secrets) via a .env file